server_vllm.py

```diff
@@ -294,7 +294,7 @@ async def wrap_vllm_generator(
                         elif isinstance(tool_choice, Tool):
                             yield tool_choice.function.name + prompt_template.fn_param_sep_token, finish_reason
                     yield delta_text, finish_reason
```

Comment:
v2 still handles this logic over here while you wrote v2.llama to handle this logic within the prompt template class itself. We should refactor v2 in a future PR. This needs some cleaning.

Reply:
yes, we will move this logic into v2, actually I havev'nt noticed this